client/my-sites/stats/pages/realtime/chart.tsx

```diff
@@ -27,16 +26,15 @@ const UPDATE_INTERVAL_IN_SECONDS = 5;
 const MINUTE_DATA_LENGTH = 30;
 
 const RealtimeChart = ( { siteId }: { siteId: number } ) => {
-	const momentInSite = useMomentInSite( siteId );
```

Comment:
Reverting those changes, as there are a few things that needs to be adjusted if we want to use site timezone here.